Makefile

```diff
@@ -2,6 +2,8 @@
 PAGES=\
 index.html\
 demo.html\
+support/support.html\
+support/commercial.html\
```

Comment:
nitpick: support/free.html and support/commercial.html would be more consistent.